CefSharp/BindingOptions.cs

```diff
@@ -30,6 +30,11 @@ public static BindingOptions DefaultBinder
         /// </summary>
         public IBinder Binder { get; set; }
 
+        /// <summary>
+        /// interceptor used for intercepting calls to methods
+        /// </summary>
+        public IInterceptor Interceptor { get; set; }
```

Comment:
Comment lacks detail, it wasn't until I looked closer at the usage example until I realized what it was for. Comment needs to be improved.